lib/MusicBrainz/Server/Edit/Alias/Edit.pm

```diff
@@ -167,10 +169,7 @@ sub initialize
     die "You must specify the alias object to edit" unless defined $alias;
     my $entity = delete $opts{entity} or die 'Missing "entity" argument';
 
-    unless (non_empty($opts{sort_name})) {
-        delete $opts{sort_name};
-        $opts{sort_name} = $opts{name} if non_empty($opts{name});
-    }
```

Comment:
I wonder if there was a reason that I don’t understand for that this code was longer than the other version. But `$opts{name}` can’t be empty (the name is a required field), so the inner condition seems redundant to me.